packages/ui/src/common/api/schemas/schema.graphql

```diff
@@ -7286,6 +7286,5 @@ input WorkingGroupWhereInput {
 }
 
 input WorkingGroupWhereUniqueInput {
-  id: ID
-  name: String
```

Comment:
At least for now, please don't change the basic schema as this is what Hydra outputs.